lib/maintenance_smelt.pm

```diff
@@ -31,8 +31,10 @@ sub get_incident_packages {
     my $mr = $_[0];
     my $gql_query = "{incidents(incidentId: $mr){edges{node{incidentpackagesSet{edges{node{package{name}}}}}}}}";
     my $graph = JSON->new->utf8->decode(query_smelt($gql_query));
-    my @nodes = @{$graph->{data}{incidents}{edges}[0]{node}{incidentpackagesSet}{edges}};
-    my @packages = map { $_->{node}{package}{name} } @nodes;
+    die "Error in getting incident data (incidentId:$mr) from SMELT" unless $graph;
+    my $nodes = $graph->{data}{incidents}{edges}[0]{node}{incidentpackagesSet}{edges};
+    die "Invalid/empty incident data (incidentId:$mr) from SMELT" unless $nodes;
+    my @packages = map { $_->{node}{package}{name} } @{$nodes};
```

Comment:
While you're at it, could you tackle: https://progress.opensuse.org/issues/134873 as a separate commit?:

PS: could you do 
```suggestion
my $exception_message = "Unexpected response code from SMELT\n"
    die $exception_message."Error in getting incident data (incidentId:$mr) from SMELT" unless $graph;
    my $nodes = $graph->{data}{incidents}{edges}[0]{node}{incidentpackagesSet}{edges};
    die $exception_message."Invalid/empty incident data (incidentId:$mr) from SMELT" unless $nodes;
    my @packages = map { $_->{node}{package}{name} } @{$nodes};
```
So that we get the auto_review work for free (see https://progress.opensuse.org/issues/134873), if you can follow up with SMELT afterwards, it would be great too!